### PR TITLE
docs: optimize

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     -      |
| 🇨🇳 Chinese |     -      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at cd40f6d</samp>

This pull request updates the `CHANGELOG.en-US.md` and `CHANGELOG.zh-CN.md` files to reflect the latest changes in the ant-design library. It organizes the changelog entries by component, removes duplicates, and adds missing items. It aims to improve the clarity and consistency of the documentation for both English and Chinese users.

### <samp>🤖 Generated by Copilot at cd40f6d</samp>

* Reorganize changelog entries by component names and add missing entries in both English and Chinese versions ([link](https://github.com/ant-design/ant-design/pull/45659/files?diff=unified&w=0#diff-e961ad288c43a2feb2d56b696e10c0b670d652dbb17c76c9aa0452d45157977cL23-R49), [link](https://github.com/ant-design/ant-design/pull/45659/files?diff=unified&w=0#diff-1224d5ca5233fdeba67ba2e2dc6fa3e615905253f35dc132c4b4b86f51f9eab4L23-R49))
* Remove duplicate changelog entries that were already grouped by component names in both English and Chinese versions ([link](https://github.com/ant-design/ant-design/pull/45659/files?diff=unified&w=0#diff-e961ad288c43a2feb2d56b696e10c0b670d652dbb17c76c9aa0452d45157977cL39-L54), [link](https://github.com/ant-design/ant-design/pull/45659/files?diff=unified&w=0#diff-1224d5ca5233fdeba67ba2e2dc6fa3e615905253f35dc132c4b4b86f51f9eab4L39-L54))

### CODE REVIEW

**Review by li-jia-nan — APPROVED** [review]

**Review by afc163 — APPROVED** [review]

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -20,12 +20,33 @@ tag: vVERSION
...
 - 🆕 Select support `optionRender` prop. [#45529](https://github.com/ant-design/ant-design/pull/45529) [@RedJue](https://github.com/RedJue)
 - 🆕 Pagination support combine `simple` and `showSizeChanger`. [#45538](https://github.com/ant-design/ant-design/pull/45538)
-- 🆕 Optimize z-index logic of popup components, and make them don't block each other by default. [#45512](https://github.com/ant-design/ant-design/pull/45512) [#45490](https://github.com/ant-design/ant-design/pull/45490) [@kiner-tang](https://github.com/kiner-tang)
+- 🆕 Spin support `fullscreen` to display as backdrop. [#44986](https://github.com/ant-design/ant-design/pull/44986) [@Rafael-Martins](https://github.com/Rafael-Martins) [#45436](https://github.com/ant-design/ant-design/pull/45436) [@li-jia-nan](https://github.com/li-jia-nan)
+- 🆕 Form `validateFields` support `dirty` for validating touched and validated fields. [#45389](https://github.com/ant-design/ant-design/pull/45389)
+- 🆕 Watermark support `inherit` prop to disable watermark pass to Drawer &amp; Modal. [#45319](https://github.com/ant-design/ant-design/pull/45319)
```
Comment (afc163) [inline CHANGELOG.en-US.md:44]: - 🆕 Watermark support `inherit` prop to disable watermark pass to Drawer and Modal. [#45319](https://github.com/ant-design/ant-design/pull/45319)

Comment (afc163) [inline CHANGELOG.en-US.md:44]:
```suggestion
- 🆕 Watermark support `inherit` prop to disable watermark pass to Drawer and Modal. [#45319](https://github.com/ant-design/ant-design/pull/45319)
```

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -20,12 +20,33 @@ tag: vVERSION
...
+- Slider
+  - 🆕 Show Slider tooltip when focus handler. [#45653](https://github.com/ant-design/ant-design/pull/45653)
+  - 💄 Slider handler should be movable after click tracker. [#45651](https://github.com/ant-design/ant-design/pull/45651)
+- InputNumber
+  - 🆕 InputNumber support `changeOnBlur` prop to disable trigger `onChange` event when blur. [#45395](https://github.com/ant-design/ant-design/pull/45395)
+  - 🐞 Fix InputNumber in Form with `hasFeedback` that will lose focus when feedback icon appear. [#45632](https://github.com/ant-design/ant-design/pull/45632) [@MadCcc](https://github.com/MadCcc)
+  - 🐞 Fix InputNumber dynamic modify `formatter` not working. [#45325](https://github.com/ant-design/ant-design/pull/45325)
+- Table
+  - 🆕 Table `columnTitle` support render function. [#41937](https://github.com/ant-design/ant-design/pull/41937) [@Zhou-Bill](https://github.com/Zhou-Bill)
+  - 🛠 🚨 Refactor Table `ref` to support `scrollTo` to scroll to target `key` or `index` or `top`. [#45245](https://github.com/ant-design/ant-design/pull/45245)
```
Comment (afc163) [inline CHANGELOG.en-US.md:32]: 多了一个 emoji

File: CHANGELOG.en-US.md [hunk 3]
```diff
@@ -20,12 +20,33 @@ tag: vVERSION
...
+- 🆕 Spin support `fullscreen` to display as backdrop. [#44986](https://github.com/ant-design/ant-design/pull/44986) [@Rafael-Martins](https://github.com/Rafael-Martins) [#45436](https://github.com/ant-design/ant-design/pull/45436) [@li-jia-nan](https://github.com/li-jia-nan)
+- 🆕 Form `validateFields` support `dirty` for validating touched and validated fields. [#45389](https://github.com/ant-design/ant-design/pull/45389)
+- 🆕 Watermark support `inherit` prop to disable watermark pass to Drawer &amp; Modal. [#45319](https://github.com/ant-design/ant-design/pull/45319)
+- 🆕 App support `component` for customization. [#45292](https://github.com/ant-design/ant-design/pull/45292)
+- 🆕 🔥 Input and Input.TextArea support `count` custom character count (for example, fix emoji character length to `1`); `count.max` supports out-of-range styles; restore emoji to native count to solve the problem of `maxLength` and `value` mismatch. [#45140](https://github.com/ant-design/ant-design/pull/45140)
```
Comment (afc163) [inline CHANGELOG.en-US.md:46]: 多了一个 emoji

File: CHANGELOG.en-US.md [hunk 4]
```diff
@@ -20,12 +20,33 @@ tag: vVERSION
 
 `2023-11-03`
 
-- 🆕 Show Slider tooltip when focus handler. [#45653](https://github.com/ant-design/ant-design/pull/45653)
-- 🆕 Slider handler should be movable after click tracker. [#45651](https://github.com/ant-design/ant-design/pull/45651)
+- Slider
+  - 🆕 Show Slider tooltip when focus handler. [#45653](https://github.com/ant-design/ant-design/pull/45653)
```
Comment (afc163) [inline CHANGELOG.en-US.md:24]:
```suggestion
  - 🆕 Slider will show tooltip when focus handler. [#45653](https://github.com/ant-design/ant-design/pull/45653)
```

**Comment by afc163** [comment]
见鬼，master 上 ci 挂了，但是我本地是好的。

File: CHANGELOG.en-US.md (hunk 2 above)
Comment (MadCcc) [inline CHANGELOG.en-US.md:32]:
```suggestion
  - 🛠 Refactor Table `ref` to support `scrollTo` to scroll to target `key` or `index` or `top`. [#45245](https://github.com/ant-design/ant-design/pull/45245)
```

File: CHANGELOG.en-US.md (hunk 3 above)
Comment (MadCcc) [inline CHANGELOG.en-US.md:46]:
```suggestion
- 🆕 Input and Input.TextArea support `count` custom character count (for example, fix emoji character length to `1`); `count.max` supports out-of-range styles; restore emoji to native count to solve the problem of `maxLength` and `value` mismatch. [#45140](https://github.com/ant-design/ant-design/pull/45140)
```

**Pull request merged**